Index: templates/cycle_angelo/profile_registration.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>{% extends 'rango/base.html' %}\r\n2 {% load staticfiles %}\r\n3\r\n4 {% block title_block %}\r\n5 Register - Step 2\r\n6 {% endblock %}\r\n7\r\n8 {% block body_block %}\r\n9 <div class=\"jumbotron p-4\">\r\n10 <div class=\"container\">\r\n11 <h1 class=\"jumbotron-heading\">Register Here - Step 2</h1>\r\n12 </div>\r\n13 </div>\r\n14\r\n15 <div class=\"container\">\r\n16 <div class=\"row\">\r\n17 <form method=\"post\"\r\n18 action=\"{% url 'cycle_angelo:register_profile' %}\"\r\n19 enctype=\"multipart/form-data\">\r\n20 {% csrf_token %}\r\n21 {{ form.as_p }}\r\n22 <input type=\"submit\" value=\"Create Profile\" />\r\n23 </form>\r\n24 </div>\r\n25 </div>\r\n26 {% endblock %}
===================================================================
diff --git a/templates/cycle_angelo/profile_registration.html b/templates/cycle_angelo/profile_registration.html
--- a/templates/cycle_angelo/profile_registration.html	
+++ b/templates/cycle_angelo/profile_registration.html	
@@ -1,4 +1,4 @@
-{% extends 'rango/base.html' %}
+{% extends 'cycle_angelo/base.html' %}
 2 {% load staticfiles %}
 3
 4 {% block title_block %}
